sklearn/metrics/tests/test_common.py

```diff
@@ -2245,6 +2258,37 @@ def check_array_api_metric_pairwise(metric, array_namespace, device, dtype_name)
     )
 
 
+PAIRWISE_METRICS = {
```

Comment:
`test_common.py` originally never checked any pairwise metrics. But we do array API checks for pairwise metrics. This is down here (and not further up with all the other metrics lists) because it's only relevant for array API stuff.